pkg/cfaws/secure_token_storage.go

```diff
@@ -28,8 +28,8 @@ func GetValidCachedToken(profileKey string) *SSOToken {
 }
 
 // Attempts to store the token, any errors will be logged to debug logging
-func StoreSSOToken(profileKey string, ssoTokenValue SSOToken) {
-	err := credstore.Store(profileKey, ssoTokenValue)
+func StoreSSOToken(profileKey string, ssoTokenValue SSOToken, profile string) {
```

Comment:
because there may be more than one "root profile" with the same sso start url, this will overwrite the description every time you need to login if you login with a different root profile